Makefile

```diff
@@ -89,26 +85,26 @@ help: ## Display this help (default)
 ## --------------------------------------
 
 .PHONY: all
-all: gomod build-all test lint ## Run all major targets (lint, test, build)
+all: gomod cross-build test lint ## Run all major targets (lint, test, cross-build)
 
 ## --------------------------------------
 ## Build
 ## --------------------------------------
 
 .PHONY: cross-build
-cross-build: ${CLI_TARGETS} ${PLUGIN_TARGETS}## Build the Tanzu Core CLI and plugins for all supported platforms
+cross-build: ${CLI_TARGETS} cross-build-publish-plugins ## Build the Tanzu Core CLI and plugins for all supported platforms
+
+.PHONY: cross-build-publish-plugins
+cross-build-publish-plugins: prepare-builder plugin-build-and-publish-packages inventory-init inventory-plugin-insert inventory-plugin-deactivate inventory-plugin-activate ## Build and publish the plugins for all supported platforms
```

Comment:
question:  what is the reason to call both `inventory-plugin-deactivate` `inventory-plugin-activate` make targets here

Reply:
Good catch. I was testing these targets and forgot to remove it. Updated.